GCRCatalogs/catalog_configs/dc2_object_run1.1p.yaml

```diff
@@ -0,0 +1,6 @@
+subclass_name: dc2_object.DC2ObjectCatalog
+base_dir: /global/projecta/projectdirs/lsst/global/in2p3/Run1.1/coadd_catalog
```

Comment:
I've taken the liberty of moving this to `object_catalog`
and putting in a symlink to preserve backward compatibility.